app/views/tasks/show.html.erb

```diff
@@ -22,7 +22,7 @@
   <% if @is_teacher %>
   <h4 align="right">
     <%= @student.full_name %> <br>
-    <%= mail_to @student.user.email, @student.user.email %>
+    <%= mail_to @student.user.additional_email, @student.user.additional_email %>
```

Comment:
So what will happen if this additional field was empty?
What will happen for existing users?